services/src/pro/datasets/in_memory.rs

```diff
@@ -902,8 +954,8 @@ mod tests {
         };
 
         let ds = AddDataset {
-            id: None,
-            name: "OgrDataset".to_string(),
+            name: None,
```

Comment:
can you add a test where the name is not None?

Reply:
https://github.com/geo-engine/geoengine/commit/6f195958364afd368eaed45db4d3c01d83cbd2e7